src/classes/ADDR_HouseholdAcc_TEST.cls

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.